### PR TITLE
Add commun

### PR DESCRIPTION
**Pull request by kokabi1365**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -9512,5 +9512,14 @@
         "domains": [
             "zynga.com"
         ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:9516]:
```suggestion

```

**Review by tupaschoal — COMMENTED** [review]
Thanks for the new entry suggestion. There are two small issues we need to tackle to solve, one of which I've annotated. The other is that the entries in the JSON are sorted.

Can you please tend to those? Thanks!

**Review by kokabi1365 — COMMENTED** [review]
I did what you said.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**